packages/react/src/components/dropdown/combobox/Combobox.tsx

```diff
@@ -83,7 +84,7 @@ export const Combobox = <OptionType,>(props: ComboboxProps<OptionType>) => {
     catchEscapeKey,
     circularNavigation = false,
     className,
-    clearable = true,
+    clearable = props.multiselect,
```

Comment:
Does this mean that the library user can not hide the clear button in the multiselect mode anymore?

Reply:
No, this is the default value. If user does not provide clearable prop, then it will use the default value.

Reply:
Ok 👍  
It might difficult to hide clearbutton in multiselect because of some linter rules 🤔